plugins/sqlite_authkey_storage_plugin/src/main/kotlin/de/busesteinkamp/plugins/auth/SqliteAuthKeyRepository.kt

```diff
@@ -0,0 +1,77 @@
+package de.busesteinkamp.plugins.auth
+
+import de.busesteinkamp.domain.auth.AuthKey
+import de.busesteinkamp.domain.auth.AuthKeyRepository
+import java.sql.DriverManager
+import java.text.SimpleDateFormat
+import java.util.*
+
+class SqliteAuthKeyRepository : AuthKeyRepository {
+    private val url = "jdbc:sqlite:auth_keys.db"
+
+    init {
+        Class.forName("org.sqlite.JDBC")
+
+        val connection = DriverManager.getConnection(url)
+        val statement = connection.createStatement()
+        statement.executeUpdate(
+            "CREATE TABLE IF NOT EXISTS auth_keys (platformName TEXT, authKey TEXT, createdAt TEXT, expiresAt TEXT)"
+        )
+        statement.close()
+        connection.close()
```

Comment:
Dies könnte eine Extra Methode sein, um das Init Übersichtlicher zu machen:
```suggestion
        createTableIfNotExists()
      }
      private fun createTableIfNotExists() {
        try (val connection = DriverManager.getConnection(url);
             val statement = connection.createStatement()) {
            statement.executeUpdate(
                "CREATE TABLE IF NOT EXISTS auth_keys (platformName TEXT PRIMARY KEY, authKey TEXT, createdAt TEXT, expiresAt TEXT)"
            )
        } catch (e: SQLException) {
            throw RuntimeException("Error creating table", e)
        }
    }
        
        
```